peachjam/js/components/FindDocuments/AdvancedSearch.vue

```diff
@@ -94,6 +94,29 @@
               >
             </div>
           </div>
+          <div class="col-12 mb-3">
```

Comment:
There's a pending change to substantially alter how advanced search works. I suggest you don't add this now, and it can be added later, once that PR is merged.

Reply:
@longhotsummer I've removed what I added in advanced search.